fastai/callbacks/tracker.py

```diff
@@ -70,7 +70,7 @@ def on_epoch_end(self, epoch, **kwargs:Any)->None:
             self.best,self.wait = current,0
         else:
             self.wait += 1
-            if self.wait > self.patience:
+            if self.wait >= self.patience:
```

Comment:
I wouldn't change this behavior as a. it's the one users know, b. it's consistent with other callbacks. We can clarify in the documentation it takes patience full epochs before applying the change at patience+1.

Reply:
Changed as suggested.